client/ayon_blender/plugins/load/load_camera_fbx.py

```diff
@@ -53,6 +53,13 @@ def _process(self, libpath, asset_group, group_name):
             obj.parent = asset_group
 
         for obj in objects:
+            if obj.name in parent.objects:
+                self.log.info(
+                    "Object with name '%s' already exists in the scene. "
+                    "Skipping linking to the scene collection.",
+                    obj.name,
+                )
+                continue
```

Comment:
Could you elaborate how this fixes it? It seems like a hacky 'fix'?

Reply:
> (as it happens when users load a camera in the same instance I guess).

Can you explain how I would reproduce this? How do I load a camera to the same instance?

Reply:
But aren't we namespacing everything? 🤔 I'm so confused.

@LiborBatek can you share a screen recording of it breaking without this PR?